Xamarin.Forms.Material.iOS/Xamarin.Forms.Material.iOS.csproj

```diff
@@ -72,6 +72,8 @@
     <ProjectReference Include="..\Xamarin.Forms.Platform.iOS\Xamarin.Forms.Platform.iOS.csproj">
       <Project>{271193C1-6E7C-429C-A36D-3F1BE5267231}</Project>
       <Name>Xamarin.Forms.Platform.iOS</Name>
+      <IsAppExtension>false</IsAppExtension>
```

Comment:
there are changes dev16 auto makes. I'm just including them here so the files stop getting touched every time we open a new project in dev16